README.md

```diff
@@ -160,16 +160,36 @@ via SMTP.
 * `suppress_empty`: if true, no email will be sent unless matching journal
 entries are found (defaults to `true`)
 
+* `from`: RFC-5322 format address to be used as the sender address (required)
+
+* `to`: RFC-5322 format address to be used as the recipient address, or a list
+of such addresses (required)
+
+* `cc`: RFC-5322 format address to be used as a carbon-copy address,
+or a list of such addresses
+
+* `bcc`: RFC-5322 format address to be used as a blind-carbon-copy address,
+or a list of such addresses
+
+* `subject`: string to be used as the email message subject
+
+* 'headers': dictionary of string keys and string values to be added as
+custom headers; the dictionary cannot include 'From', 'To', 'Cc', or 'Bcc'
+
+Either command-based or SMTP-based delivery must be specified (but not both).
+
 ### Email via command
 
 Example:
 ```yaml
 email:
-  command: "mail -s 'journal output' admin@example.com"
+  from: "journal sender" <journal@example.com>
+  to: "system admin" <admin@example.com>
+  command: "sendmail -i -t"
 ```
 
 This will cause journal-brief to execute the specified command in a
-child process and pipe the formatted journal entries to it. The supplied
+child process and pipe the formatted email message to it. The supplied
```

Comment:
Given this semantic change, I wonder if we should change the name (e.g. from 'command' to 'pipe'). Although, if anyone uses the old command line for this the worst that will happen is they'll get headers in the email body.

What do you think?

Reply:
I'm planning on adding backwards-compatibility, so that existing configurations won't be broken, but the non-MIME 'command' mode will no longer be documented. I'm certainly open to other options, but since the blog post got published today I think I need to at least continue supporting the configurations included in it :-)

Reply:
Oh, it's published, great!